AUTHORS

```diff
@@ -999,12 +998,55 @@ Shubham Agrawal <shubham.ag6845@gmail.com>
 numbermaniac <5206120+numbermaniac@users.noreply.github.com>
 Sakirul Alam <binarysakir@gmail.com>
 Mohammed Bilal <r.mohammedbilal@gmail.com>
+Naveen Sai <naveensaisreenivas@gmail.com>
 Chris du Plessis <christopherjonduplessis@gmail.com>
 Coder-RG <rgoel1999@gmail.com>
 Ansh Mishra <anshmishra471@gmail.com>
 Alex Malins <github@alexmalins.com>
 Lorenzo Contento <lorenzo.contento@gmail.com>
-Naveen Sai <naveensaisreenivas@gmail.com>
 Shital Mule <shitalmule04@gmail.com>
 Amanda Dsouza <meezamanda@yahoo.com>
 Nijso Beishuizen <nijso@koolmees.numerically-related.com>
+Harry Zheng <harry@harryzheng.com>
+Felix Yan <felixonmars@archlinux.org>
+Constantin Mateescu <costica1234@me.com>
+Eva Tiwari <eva.tiwari@gmail.com>
+Aditya Kumar Sinha <adityakumar113141@gmail.com>
+Soumi Bardhan <51290447+Soumi7@users.noreply.github.com>
+Kaustubh Chaudhari <ckaustubhm06@gmail.com>
+Kristian Brünn <hello@kristianbrunn.com>
+Neel Gorasiya <mgorasiya1974@gmail.com>
+Akshat Sood <68052998+akshatsood2249@users.noreply.github.com>
+Jose M. Gomez <chemoki@gmail.com>
+Stefan Petrea <stefan@garage-coding.com>
+Praveen Sahu <povinsahu@gmail.com>
+Mark Bell <mark00bell@googlemail.com>
+AlexCQY <alex_chua@u.nus.edu>
+Fabian Froehlich <fabian@schaluck.com>
+Nikhil Gopalam <gopalamn@umich.edu>
+Kartik Sethi <kartiks31416@gmail.com>
+MaqOwais <quadirowais200@gmail.com>
```

Comment:
@MaqOwais how should your name be recorded here?

Reply:
You may use my full name as **Muhammed Abdul Quadir Owais** or if there is a length of character constraint _MaqOwais_ is Ok.

Reply:
Thanks. Updated.